pkg/datapath/linux/node_linux_test.go

```diff
@@ -1405,7 +1405,7 @@ func (s *linuxPrivilegedIPv4OnlyTestSuite) TestArpPingHandling(c *check.C) {
 	c.Assert(found, check.Equals, false)
 
 	c.Assert(linuxNodeHandler.NodeAdd(nodev3), check.IsNil)
-	time.Sleep(100 * time.Millisecond) // insertNeighbor is invoked async
+	wait(nodev3.Identity(), nil, false)
```

Comment:
Either you need to pass the `now` value or make the function `wait` to accept `before = nil`.